### PR TITLE
Initial Dependency Manager docs

### PR DESCRIPTION
**Pull request by jasondaming**

### CODE REVIEW

**Comment by sciencewhiz** [comment]
Add links in the VS Code section of the changelog, and in the beta task

File: source/docs/software/vscode-overview/3rd-party-libraries.rst [hunk 1]
```diff
@@ -14,29 +14,42 @@ Vendor dependencies are installed on a per-project basis (so each robot project
...
 
-This JSON file allows specification of complex libraries with multiple components (Java, C++, JNI, etc.) and also helps handle some complexities related to simulation. Vendors that choose to provide a remote URL in the JSON also enable users to check for updates from within VS Code.
+      **VS Code**
 
-#### How Does It Work? - LabVIEW
+      .. image:: images/3rd-party-libraries/dependency-activity-bar.png
+         :alt: The activity bar of VS Code showing the WPILib icon that opens the Dependency Manager.
 
-For LabVIEW teams, there might be a few new :guilabel:`Third Party` items on various palettes (specifically, one in :guilabel:`Actuators`, one in :guilabel:`Actuators` -> :guilabel:`Motor Control` labeled :guilabel:`CAN Motor`, and one in :guilabel:`Sensors`). These correspond to folders in ``C:\Program Files\National Instruments\LabVIEW 2023\vi.lib\Rock Robotics\WPI\Third Party``
+      All vendordep operations can be controlled by the Dependency Manager.  Click the button shown above to access the interface.
```
Comment (sciencewhiz) [inline source/docs/software/vscode-overview/3rd-party-libraries.rst:29]:
```suggestion
      All vendordep operations can be controlled by the Dependency Manager.  Click the WPILib logo in the sidebar as shown above to access the interface.
```

File: source/docs/software/vscode-overview/3rd-party-libraries.rst [hunk 2]
```diff
@@ -72,221 +84,20 @@ Third party libraries are packaged into Python wheels and uploaded to PyPI (if p
 
       The ``vendordep`` gradle task can also fetch vendordep JSONs from the user ``wpilib`` folder. To do so, pass ``FRCLOCAL/Filename.json`` as the file URL. For example, ``gradlew vendordep --url=FRCLOCAL/WPILibNewCommands.json`` will fetch the JSON for the command-based framework.
```
Comment (sciencewhiz) [inline source/docs/software/vscode-overview/3rd-party-libraries.rst:85]: This Command-Line section feels like it should be another tab, rather then hidden in the legacy VS Code section

File: source/docs/software/vscode-overview/3rd-party-libraries.rst [hunk 3]
```diff
@@ -72,221 +84,20 @@ Third party libraries are packaged into Python wheels and uploaded to PyPI (if p
...
-
-[ChoreoLib](https://sleipnirgroup.github.io/Choreo/choreolib/installation/) - Library for reading and following trajectories generated by [Choreo](https://sleipnirgroup.github.io/Choreo/)
-
-.. tab-set::
-
-   .. tab-item:: Java/C++
-      :sync: javacpp
-
-      ``https://sleipnirgroup.github.io/ChoreoLib/dep/ChoreoLib.json``
+### How Does It Work?
```
Comment (sciencewhiz) [inline source/docs/software/vscode-overview/3rd-party-libraries.rst:87]: This feels like it can be tabs as well

Comment (jasondaming) [reply source/docs/software/vscode-overview/3rd-party-libraries.rst:87]: I disagree that How does it work should go in tabs.  It is critical for placing offline install files and really has nothing to do with the install.  All of the other tabs are direct ways to install a vendordep

Comment (sciencewhiz) [reply source/docs/software/vscode-overview/3rd-party-libraries.rst:87]: There's 3 sections for different languages that you don't need to read unless you're using that language. Isn't that what we use tabs for?

Comment (jasondaming) [reply source/docs/software/vscode-overview/3rd-party-libraries.rst:87]: I understand now you wanted to add a new set of tabs for it.  I thought you meant adding a tab to the one above it.

I will add the tab to it.

**Review by sciencewhiz — APPROVED** [review]

**Pull request merged**